documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+```shell
+mongosh mongodb://<username>:<password>@<host>/<database name>?replicaSet=<replica set name>
+```
+
+You should now see:
+```
+Current Mongosh Log ID:	6389e2adec352d5447551ae0
+Connecting to:		mongodb://<credentials>@10.23.62.156/admin?replicaSet=mongodb&appName=mongosh+1.6.1
+Using MongoDB:		5.0.14
+Using Mongosh:		1.6.1
+
+For mongosh info see: https://docs.mongodb.com/mongodb-shell/
+
+
+To help improve our products, anonymous usage data is collected and sent to MongoDB periodically (https://www.mongodb.com/legal/privacy-policy).
+You can opt-out by running the disableTelemetry() command.
+
+------
+   The server generated these startup warnings when booting
+   2022-12-02T11:24:05.416+00:00: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine. See http://dochub.mongodb.org/core/prodnotes-filesystem
+------
+
+------
+   Enable MongoDB's free cloud-based monitoring service, which will then receive and display
+   metrics about your deployment (disk utilization, CPU, operation statistics, etc).
+
+   The monitoring data will be available on a MongoDB website with a unique URL accessible to you
+   and anyone you share the URL with. MongoDB may use this information to make product
+   improvements and to suggest MongoDB products and deployment options to you.
+
+   To enable free monitoring, run the following command: db.enableFreeMonitoring()
+   To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
+------
+
+mongodb [primary] admin>
+```
+
+You can now interact with MongoDB directly using any [MongoDB commands](https://www.mongodb.com/docs/manual/reference/command/). For example entering `show dbs` should output something like:
+```
+admin   172.00 KiB
+config  120.00 KiB
+local   404.00 KiB
+```
+Now that we have access to MongoDB we can create a database named `test-db`. To create this database enter:
+```shell
+use test-db
+```
+Now lets create a user called `testUser` read/write access to the database `test-db` that we just created . Enter:
+```shell
+db.createUser({
+  user: "testUser",
+  pwd: "password",
+  roles: [
+    { role: "readWrite", db: "test-db" }
+  ]
+})
+```
+You can verify that you added the user correctly, by entering the command `show users` into the mongo shell, this will output:
+```json
+[
+  {
+    _id: 'test-db.testUser',
+    userId: new UUID("6e841e28-b1bc-4719-bf42-ba4b164fc546"),
+    user: 'testUser',
+    db: 'test-db',
+    roles: [ { role: 'readWrite', db: 'test-db' } ],
+    mechanisms: [ 'SCRAM-SHA-1', 'SCRAM-SHA-256' ]
+  }
+]
+```
+Feel free to test out any other MongoDB commands, when you’re ready to leave the MongoDB shell you can just type `exit`. Once you've entered `exit`, you should be back in the shell you started in where you can interact with Juju and LXD.
```

Comment:
```suggestion
Feel free to test out any other MongoDB commands. When you’re ready to leave the MongoDB shell you can just type `exit`. Once you've typed `exit` you will be back in the original shell, where you can interact with Juju and LXD.
```

Reply:
I suggest these changes for clarity - saying "entered 'exit'" is a slightly odd phrasing.